app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">JH Mental Helper</string>
+    <string-array name="genders">
+        <item>Gender</item>
+        <item>Female</item>
```

Comment:
I guess you could add `Unmentioned` field here, too, after my PR is approved. Yeah sorry, the updated version of `Gender` is still pending on request.